### PR TITLE
add product-range screen

### PR DESCRIPTION
**Pull request by harshkotkar5557**

### CODE REVIEW

**Review by vijaytembugade — APPROVED** [review]

File: product-range/product-range.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    height: 85vh;
+    overflow: auto;
+    scroll-margin-top: 1rem;
+    padding: 1rem;
+}
+.product-container::-webkit-scrollbar { 
+    display: none;  /* Safari and Chrome */
+}
+.shadow{
+    width: 200px;
```
Comment (viru2001) [inline product-range/product-range.css:58]: you can use relative units here

File: product-range/product-range.html [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobby</title>
+    <link rel="stylesheet" 
+		href="https://kaizen-ui.netlify.app//styles.css">
+    <link rel="stylesheet" href="../style.css">
+    <link rel="stylesheet" 
+	  href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (viru2001) [inline product-range/product-range.html:12]: you can import this in css

**Review by viru2001 — COMMENTED** [review]
I gave some suggestions. everything else is good

**Review by MuskanChhatrasal — COMMENTED** [review]
Try using rem values. It will be helpful while adding responsiveness.
Else LGTM!!

**Pull request merged**